src/contributing/void-docs/submitting.md

```diff
@@ -0,0 +1,51 @@
+## Submitting Changes
+
+### Requirements
+
+To clone the repository and push changes
+[git(1)](https://man.voidlinux.org/git.1) is required, which is available as the
+`git` package.
+
+Building the Void Handbook locally requires
+[mdBook](https://rust-lang-nursery.github.io/mdBook/), which can be installed
+with the `mdBook` package on Void. At the root of the void-docs repository
+`mdbook serve` can be run to serve the docs on your localhost.
+
+### Forking
+
+To fork the repository a [github account](https://github.com/join) is needed.
+After the account is created follow github's
+[guide](https://help.github.com/en/articles/fork-a-repo) on setting up a fork.
+
+Clone the repository onto your computer, enter it, and create a new branch:
+
+```
+$ git clone https://github.com/YOUR_USERNAME/void-docs.git
+$ cd void-docs
+$ git checkout -b <BRANCH_NAME>
+```
+
+> The branch name should be based off of the planned change.
```

Comment:
I think this can be removed because the branch name can be arbitrary. It literally doesn't matter at all in the process what the branch is named on your fork.

Reply:
Removed that bit.